pokemongo_bot/cell_workers/pokemon_catch_worker.py

```diff
@@ -10,9 +10,9 @@
 
 def get_pokemon_ids_from_inventory(response_dict):
     id_list = []
-    inventory_items = response_dict.get("responses", {}).get("GET_INVENTORY", {}).get("inventory_delta", {}).get(
-        "inventory_items")
     if response_dict is not None:
+        inventory_items = response_dict.get("responses", {}).get("GET_INVENTORY", {}).get("inventory_delta", {}).get(
+            "inventory_items")
```

Comment:
This handles KeyError in a nicer way. If it doesn't have the property, then it will return None or the second argument. Also, I don't think we can assume the API returns any of the properties.